pkg/work/spoke/apply/server_side_apply.go

```diff
@@ -72,3 +80,27 @@ func (c *ServerSideApply) Apply(
 
 	return obj, err
 }
+
+func removeCreationTimeFromMetadata(obj map[string]interface{}) {
+	if metadata, found := obj["metadata"]; found {
+		if metaObj, ok := metadata.(map[string]interface{}); ok {
+			klog.V(4).Infof("remove `metadata.creationTimestamp`")
```

Comment:
should we also check the value of creationTimestamp is nil

Reply:
we should

Reply:
+1 to @qiujian16 comment.